src/bindings/compat/java/src/cvc3/ValidityChecker.java

```diff
@@ -919,47 +919,47 @@ public OpMut createOp(String name, Type type, Expr expr) throws Cvc3Exception {
 
     // '='
     public OpMut eqOp() throws Cvc3Exception {
-	return new OpMut(jniEqOp(), embeddedManager());
+	return new OpMut(jniEqOp(embedded()), embeddedManager());
```

Comment:
I tried almost the exact same changes to jniDivideOp, etc. and ValidityChecker_impl.cpp and the code generator complained. I'm not sure why this version compiles and what I tried did not :-/

The only difference is these eqOp, ltOp, etc. changes. How did you figure out that you needed this?

Reply:
Pattern matching :) My changes are based on what [jniGetExprManager](https://github.com/CVC4/CVC4/blob/master/src/bindings/compat/java/src/cvc3/ValidityChecker_impl.cpp#L127-L129) is doing and the reason why I added an argument of type `ValidityChecker` to jniEqOp/jniLtOp/etc. (instead of `ExprManager` directly) was that all other functions in ValidityChecker seem to do the same. To get there, I followed the compiler errors.